Functions.Templates/Templates/IoTHubTrigger-TypeScript/index.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"index.js","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":";;AAAA,aAAoB,OAAY,EAAE,cAAqB;IACnD,OAAO,CAAC,GAAG,CAAC,mEAAiE,cAAgB,CAAC,CAAC;IAE/F,cAAc,CAAC,OAAO,CAAC,UAAA,OAAO;QAC1B,OAAO,CAAC,GAAG,CAAC,uBAAqB,OAAS,CAAC,CAAC;IAChD,CAAC,CAAC,CAAC;IAEH,OAAO,CAAC,IAAI,EAAE,CAAC;AACnB,CAAC;AARD,kBAQC;AAAA,CAAC"}
```

Comment:
deleted .js and .js.map


---
In reply to: [133859668](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859668) [](ancestors = 133859668)